tests/python/pants_test/backend/python/tasks2/test_python_distribution_integration.py

```diff
@@ -17,69 +17,61 @@ class PythonDistributionIntegrationTest(PantsRunIntegrationTest):
   # whl by setup.py) and an associated test to be consumed by the pants goals tested below.
   superhello_project = 'examples/src/python/example/python_distribution/hello/superhello'
   superhello_tests = 'examples/tests/python/example/python_distribution/hello/test_superhello'
+  superhello_testprojects_project = 'testprojects/src/python/python_targets/python_distribution/superhello'
 
   def test_pants_binary(self):
-    self._maybe_test_pants_binary('2.7')
+    command=['binary', '{}:main'.format(self.superhello_project)]
+    pants_run_27 = self.run_pants(command=command)
```

Comment:
Since its assumed that pants is running 2.7, is it necessary to specify that in these variables? Or are you future-proofing a bit?

Reply:
Def not necessary, I can clean them up.